### PR TITLE
Created a simple "special edition" for testing

### PR DESCRIPTION
**Pull request by blishen**
## What's changed?
Added a special edition template
Identified it to the app in `app/model/editions/EditionsTemplates.scala`

## Implementation notes
All edited in Github by a product manager, Justin better give this a thumbs up or ideally test in CODE?

### CODE REVIEW

**Comment by blishen** [comment]
@aug24 Could you check this out? @jimhunty and I would like to see what happens - ideally in CODE. We know we're short of a couple of fields, the two commented out in the template: `expiry` and `image`

File: app/model/editions/templates/TestSpecialEdition.scala
Comment (aug24) [inline app/model/editions/templates/TestSpecialEdition.scala:33]:
```suggestion
  def Front2 = front("Front 2", None,
```

Comment (blishen) [reply app/model/editions/templates/TestSpecialEdition.scala:33]: Thanks!

**Comment by blishen** [comment]
@aug24 @philmcmahon Do we need to test this in CODE before merging into PROD?

**Comment by philmcmahon** [comment]
This now depends on https://github.com/guardian/facia-tool/pull/1241 following my [most recent commit](https://github.com/guardian/facia-tool/pull/1239/commits/0ee2f6370a8f7da8545708262751699f47152142) - which adds a load of special edition fields such as expiry, image etc.

**Review by aug24 — APPROVED** [review]

**Comment by philmcmahon** [comment]
Note: if we merge  this it will launch a special edition! So do not merge!

**Comment by philmcmahon** [comment]
Closing this as it is replaced by https://github.com/guardian/facia-tool/pull/1256

**Pull request closed without merging**
